src/pages/q/[[...all]].tsx

```diff
@@ -211,6 +212,13 @@ export const getServerSideProps: GetServerSideProps = async function ({
     )
     try {
       initialInstructor = await loadInstructor(instructorSlug)
+
+      sanityInstructor = await loadSanityInstructor(instructorSlug)
+
+      initialInstructor = {
+        ...initialInstructor,
+        ...sanityInstructor,
+      }
```

Comment:
None of the data is being overridden but it makes sense to me that sanity data would take precedence over the `initialInstructor` data

Reply:
I think this pattern makes a lot of sense so we don't have two+ data sources at the component, we just bring it together at the server level.

Reply:
since we are using a `stale-while-revalidate` cache header, it should be performant!

Reply:
I agree, it was also super easy to refactor